WcaOnRails/app/models/competition.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
I just tried removing the .freeze on line 52 but rubocop wasn't happy :)

`
Offenses:
app/models/competition.rb:53:26: C: Freeze mutable objects assigned to constants.
  INVALID_NAME_MESSAGE = "must end with a year and must contain only alphnumeric characters, dashes(-), ampersands(&), periods(.), colons(:), apostrophes('), and spaces( )"
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^`